pom.xml

```diff
@@ -111,14 +123,63 @@
             <scope>runtime</scope>
         </dependency>
         <dependency>
-            <!--
-            Include all of the production JARs without observability fixes.
-            See https://www.oracle.com/database/technologies/maven-central-guide.html
-            -->
             <groupId>com.oracle.database.jdbc</groupId>
-            <artifactId>ojdbc8-production</artifactId>
+            <artifactId>ojdbc8</artifactId>
+            <version>${oracle.jdbc.driver.version}</version>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.oracle.database.jdbc</groupId>
+            <artifactId>ucp</artifactId>
+            <version>${oracle.jdbc.driver.version}</version>
```

Comment:
nit: we can avoid the version here? (applies to all dependent runtime libs brought by ojdbc-bom)

Reply:
@anupamaggarwal Adding version here for Fossa because it was picking up latest package if version was not present. I'll comment this too

Reply:
@Tanish0019  didn't understand this fully , are you saying fossa requires an explicit version (and is not happy if it picks the versions from the bom in the dependency mgmt section)?

Reply:
If I don't supply a version fossa will pick up the latest version in its scan. I have currently manually set these dependencies to be ignored by fossa scan and given the link to their license in the comment there. I am not sure if that ignore will work if version is changed so added explicitly over here

Reply:
@Tanish0019  - I se, in that case we can probably remove the dependency management section? (where we have the bom imported in pom scope), it would be redundant since we are pinning the versions explicitly.

Reply:
@anupamaggarwal updated